pkg/cedar/intake/README.md

```diff
@@ -3,6 +3,14 @@
 This folder contains code for submitting data to the CEDAR Intake API for eventual storage in Alfabet. 
 The Intake API is designed to take a JSON string in its `Body` field, as well as a schema identifier in its `Schema` field.
 
+## Swagger
+
+The Swagger file for the Intake API is saved as `cedar_intake.json` in this folder. The Swagger we get from CEDAR needs preprocessing to add the `"x-nullable": true` property to the definition of the `clientLastUpdatedDate` field on the `IntakeInput` object. This is an optional field, but without the `"x-nullable": true` property, go-swagger will generate the corresponding Go code with a `strfmt.DateTime` field. If this type is used and `.ClientLastUpdatedDate` is not set in our Go code, the zero value of `0001-01-01T00:00:00.000Z` will get sent to CEDAR Intake, which causes an error. With `"x-nullable": true` set, go-swagger generates Go code with a `*strfmt.DateTime` pointer field, and a `nil` pointer will prevent this field from being sent to CEDAR.
```

Comment:
Should this be something we ask the CEDAR team to change so we don't have to have a workaround for? It stands out as a bit strange that `clientLastUpdatedDate` is the only nullable field

Also - it's possible for us to have a `NULL` updatedAt in our DB? Maybe if we made that field not nullable and set it where appropriate we could not need this specific fix 🤔

Reply:
I'm not sure that `clientLastUpdatedDate` being nullable is necessarily weird? Doesn't seem weird enough to be worth asking the CEDAR team to rework, honestly.

As for updatedAt columns being nullable:
* business_cases.updated_at is nullable, the corresponding field in pkg/models/business_case.go is a pointer
* grt_feedback.updated_at is not nullable, but the corresponding field in pkg/models/grt_feedback.go is a pointer
* system_intakes.updated_at is nullable, the corresponding field in pkg/models/system_intake.go is a pointer.
I feel like it'd take a decent bit of rework to make the DB fields all non-nullable, as well as some work to change types on the Go fields.

Reply:
I could see the paradigm either way for either setting the updated_at columns nullable or not. I could imagine just leaving it null for insert, but it would be problematic for it to be null when it was updated. Could maybe do something with DEFAULT CURRENT_TIMESTAMP. 

Experimented with this a little bit yesterday, I think that it works slightly different in POSTGRES than MSSQL.

Reply:
Yeah - I think for the scope of this work we leave it as is, but going forward might be nice to make our created/updated fields non-null, just as a nice to have (and would eventually simplify this logic, AFAIK)

Reply:
Sounds good. We've got [EASI-1693](https://jiraent.cms.gov/browse/EASI-1693) to cover future work.